ocfweb/templates/base.html

```diff
@@ -101,7 +101,8 @@
             <div class="container">
                 <p>
                     <strong>Vote for the Student Tech Fund's renewal during the ASUC Elections @ April 5-7!</strong>
-                    <span class="nowrap">
+                    <span>
+                        <br>
```

Comment:
Looks good, seems like a pretty quick fix that'll make the banner look nice on mobile. Thanks for doing this!